### PR TITLE
(MAINT) Misc updates to project sync script

### PR DESCRIPTION
**Pull request by scotje**
Map a few more JIRA statuses to "in progress" column and broaden the project name filter to pick up our new release name format.

### CODE REVIEW

**Comment by scotje** [comment]
Executed locally and it seemed to update https://github.com/puppetlabs/pdk-planning/projects/8 correctly.

**Comment by scotje** [comment]
Not sure why the appveyor job doesn't pick up the configured ENV vars...

**Comment by glennsarti** [comment]
It shouldn't be running anyway.

**Comment by glennsarti** [comment]
Secure ENV Vars don't appear in PRs, only on branch building.

**Comment by scotje** [comment]
Gotcha, that makes sense.

File: tools/SyncProjects.ps1 [hunk 1]
```diff
@@ -163,6 +163,10 @@ $($issue.Summary)
       }
       # There's probably more than this for in progress...
       if ($Issue.Status -eq 'In progress') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for merge') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for test') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for ci') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for review') { $hash['expectedColumn'] = 'In progress'}
       if ($Issue.Status -eq 'Closed') { $hash['expectedColumn'] = 'Done'}
       if ($Issue.Status -eq 'Resolved') { $hash['expectedColumn'] = 'Done'}
       if ($Issue.Status -eq 'Completed') { $hash['expectedColumn'] = 'Done'}
```
Comment (rodjek) [inline tools/SyncProjects.ps1]: This block of conditionals should probably just be turned into a switch statement
```powershell
$hash['expectedColumn'] = switch($Issue.Status)
{
  'In progress' { 'In progress' }
  'Ready for merge' { 'In progress' }
  # &c
}
```

Comment (glennsarti) [reply tools/SyncProjects.ps1]: `switch` doesn't return a value, it executes a block

**Pull request merged**